src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -187,6 +206,151 @@ protected override IEnumerable<Model.AzureSqlManagedInstanceModel> GetEntity()
             return new List<AzureSqlManagedInstanceModel>() { ModelAdapter.GetManagedInstance(this.ResourceGroupName, this.Name) };
         }
 
+        /// <summary>
+        /// Validate requested edition.
+        /// </summary>
+        protected void ValidateRequestedEdition()
+        {
+            ModelAdapter = InitModelAdapter();
+            AzureSqlManagedInstanceModel existingInstance = ModelAdapter.GetManagedInstance(this.ResourceGroupName, this.Name);
+            Management.Internal.Resources.Models.Sku Sku = new Management.Internal.Resources.Models.Sku();
```

Comment:
Not used

Reply:
It is used in ExecuteCmdlet() method:

`            // Validate requested edition
            if (!string.IsNullOrWhiteSpace(this.SkuName) || !string.IsNullOrWhiteSpace(this.Edition))
            {
                this.ValidateRequestedEdition();
            }`

Reply:
Great catch. Thanks. Fixed